client-js/README.md

```diff
@@ -0,0 +1,202 @@
+      collectionSymbol: "MY-NFTS",
+      totalTokenSupply: "1000",
+      ownershipMode: NFTOwnershipMode.Transferable,
+      nftKind: NFTKind.Physical,
+      jsonSchema: {
+        properties: {
+          color: { name: "color", description: "", required: true },
+          size: { name: "size", description: "", required: true },
+          material: { name: "material", description: "", required: true },
+          condition: { name: "condition", description: "", required: false },
+        },
+      },
+      nftMetadataKind: NFTMetadataKind.CustomValidated,
+      identifierMode: NFTIdentifierMode.Ordinal,
+      metadataMutability: MetadataMutability.Immutable,
+      mintingMode: MintingMode.Installer,
+    },
+    "165000000000",
+    FAUCET_KEYS.publicKey,
+    [FAUCET_KEYS]
+  );
+
+  const hash = await installDeploy.send(process.env.NODE_URL!);
+
+```
+
+`InstallArgs` are specified in the associated `.env` file as follows:
+
+* `collectionName` - The name of the NFT collection, passed in as a `String`. This parameter is required and cannot be changed post installation.
+
+* `collectionSymbol` - The symbol representing a given NFT collection, passed in as a `String`. This parameter is required and cannot be changed post installation.
+
+* `totalTokenSupply` - The total number of NFTs that a specific instance of a contract will mint passed in as a `U64` value. This parameter is required and cannot be changed post installation.
+
+* `ownershipMode` - The `OwnershipMode` modality that dictates the ownership behavior of the NFT contract. This argument is passed in as a `u8` value and is required at the time of installation.
+
+* `nftKind` - The `NFTKind` modality that specifies the off-chain items represented by the on-chain NFT data. This argument is passed in as a `u8` value and is required at the time of installation.
+
+* `jsonSchema` - The JSON schema for the NFT tokens that will be minted by the NFT contract passed in as a `String`. This parameter is required if the metadata kind is set to `CustomValidated(4)` and cannot be changed post installation.
+
+* `nftMetadataKind` - The metadata schema for the NFTs to be minted by the NFT contract. This argument is passed in as a `u8` value and is required at the time of installation.
+
+* `identifierMode` - The `NFTIdentifierMode` modality dictates the primary identifier for NFTs minted by the contract. This argument is passed in as a `u8` value and is required at the time of installation.
+
+* `metadataMutability` - The `MetadataMutability` modality dictates whether the metadata of minted NFTs can be updated. This argument is passed in as a `u8` value and is required at the time of installation.
+
+* `mintingmode?` - The `MintingMode` modality that dictates the access to the `mint()` entry-point in the NFT contract. This is an optional parameter that will default to restricting access to the installer of the contract. This parameter cannot be changed once the contract has been installed.
```

Comment:
```suggestion
* `mintingmode?` - The `MintingMode` modality dictates the access to the `mint()` entry point in the NFT contract. This optional parameter will default to restricting access to the installer of the contract. **This parameter cannot be changed once the contract has been installed**.
```

Reply:
Should the formatting be: mintingMode (without a question mark)?

Reply:
I could see it going either way, if only to indicate that it's optional. I do outline that it's optional in the text definition, so I'm fine removing the question mark.

Reply:
I understand, ok, either way then :)